pyuvsim/tests/test_simsetup.py

```diff
@@ -576,6 +579,7 @@ def test_param_reader():
     assert hera_uv.blt_order != uv_obj.blt_order
     hera_uv.reorder_blts("time", "baseline")
 
+    uv_obj._consolidate_phase_center_catalogs(other=hera_uv, ignore_name=True)
```

Comment:
what does this do?

Reply:
It renumbers and/or renames the phase center catalogs so that the equality passes. It's basically a convenience for testing.